vpr/src/route/rr_graph_reader.cpp

```diff
@@ -701,7 +708,6 @@ void process_rr_node_indices(const DeviceGrid& grid) {
     /*Add the correct node into the vector
      * Note that CHANX and CHANY 's x and y are swapped due to the chan and seg convention.
```

Comment:
This comment is out of date; should be updated to say we're counting the maximum ptc_num at any location and then allocated enough storage for the indices lookup data structure.